satpy/FY4A_agri_introduction(EN).ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>Are there any dependencies that aren't instead with satpy by default that are needed to run this tutorial ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Maybe <code>cartopy</code>.  I will test it in a new environment.</p>

Reply:
<p>No, <code>conda install -c conda-forge satpy</code> is enough.</p>